### PR TITLE
Documentation: Translate to English (WIP)

### PR DESCRIPTION
**Pull request by dantleech**
I made a start on this about 9 months ago but never got round to finishing it.

### CODE REVIEW

File: Documentation/En/Index.xyl [hunk 1]
```diff
@@ -3,8 +3,156 @@
...
+  <code>Hoa\Worker</code> takes advantage of this idea, once the zombie is
+  created, to create a <strong>server</strong> which will receive
+  <strong>messages</strong>. For each message recieved, an
+  <strong>event</strong> will be emitted that we can intercept, and with
+  which we can execute a <strong>task</strong>.</p>
+
+  <p>Examples of tasks could be: sending a notification (for example, by mail),
+  executing important operations on the database, to process multiple files
+  etc. The idea is to <strong>unload</strong> from the application tasks which
+  could make is <strong>slower</strong> and execute them in
```
Comment (dantleech) [inline Documentation/En/Index.xyl]: s/is/it

File: Documentation/En/Index.xyl [hunk 2]
```diff
@@ -3,8 +3,156 @@
...
+  <strong>parallel</strong> without interfering with the execution of the
+  application.</p>
+
+<p>Even if the established protocal permits a level of <strong>integrity</strong> and <strong>reliablity</strong> of the messages, it is preferable that the commiunications happen <strong>server to server</strong> and are not routed through the client.
```
Comment (dantleech) [inline Documentation/En/Index.xyl]: s/protocal/protocol

**Comment by Hywan** [comment]
@osaris Is it OK for you to be the assignee?

**Comment by dantleech** [comment]
Oops, just finished and rebased only to realise that this has now been translated! Should have made the PR 8 months ago.

Btw have you ever considered [restructured text](https://en.wikipedia.org/wiki/ReStructuredText) and [sphinx](http://www.sphinx-doc.org/en/stable/rest.html). It is used in the Symfony and Doctrine projects, and I would vouch for it being easier to work with than XML (its also easy to host on [readthedocs](https://readthedocs.org)).

**Comment by Hywan** [comment]
Why did you close the PR?

**Comment by dantleech** [comment]
@Hywan as noted above, this work was already [done](https://github.com/hoaproject/Worker/blob/master/Documentation/En/Index.xyl). :)

**Comment by Hywan** [comment]
:+&:

**Comment by Hywan** [comment]
by @osaris

**Pull request closed without merging**